iree/compiler/Conversion/LinalgToSPIRV/SplitDispatchFunctionPass.cpp

```diff
@@ -153,7 +153,8 @@ LogicalResult SplitDispatchFunctionPass::splitDispatchFunction(
   // Collect all Linalg and scf.parallel ops for distributing.
   SmallVector<Operation *, 4> separableOps;
   for (Operation &op : fnBody)
-    if (isa<linalg::LinalgOp>(op) || isa<scf::ParallelOp>(op))
+    if (isa<linalg::LinalgOp>(op) || isa<scf::ParallelOp>(op) ||
+        isa<scf::ForOp>(op))
```

Comment:
I think we can use `isa<linalg::LinalgOp, scf::ParallelOp, scf::ForOp>(op)` now.